### PR TITLE
Add tabsize and eol

### PR DESCRIPTION
**Pull request by LaurentLesle**
Fixes: #85 

image

### CODE REVIEW

**Review by lonegunmanb — APPROVED** [review]
LGTM @LaurentLesle , WDYT @matt-FFFFFF ?

**Comment by matt-FFFFFF** [comment]
In the bicep variant they use editorconfig. There's a VScode extension for this.

Also we should consider .gitattributes as well to force unix line endings.

**Comment by segraef** [comment]
> In the bicep variant they use editorconfig. There's a VScode extension for this.
> 
> Also we should consider .gitattributes as well to force unix line endings.

Yes please include 

- https://github.com/Azure/bicep-registry-modules/blob/main/.editorconfig

and a `.gitattributes` with the following *suggested* content

```
*.tfvars  text eol=lf
*hcl text eol=lf
*.go text eol=lf
*.tf text eol=lf
```

**Comment by segraef** [comment]
> > In the bicep variant they use editorconfig. There's a VScode extension for this.
> > Also we should consider .gitattributes as well to force unix line endings.
> 
> Yes please include
> 
> * https://github.com/Azure/bicep-registry-modules/blob/main/.editorconfig
> 
> and a `.gitattributes` with the following _suggested_ content
> 
> ```
> *.tfvars text eol=lf
> *hcl text eol=lf
> *.go text eol=lf
> *.tf text eol=lf
> ```
> 
> But I also agree to simply just having the suggested [Workspace settings.json](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settingsjson-location) from @LaurentLesle because no extension is required and VSCode will inherit it right away.

@LaurentLesle any update on this one?

**Comment by segraef** [comment]
@LaurentLesle how are we doing with this one? Do you need any support to include all changes?

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*.tfvars text eol=lf
+*hcl text eol=lf
```
Comment (segraef) [inline .gitattributes:2]:
```suggestion
*.hcl text eol=lf
```

**Review by segraef — CHANGES_REQUESTED** [review]

**Review by segraef — APPROVED** [review]
🥇

**Pull request merged**